deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -81,14 +81,9 @@ objects:
           - "--cluster_name=hawkular-metrics"
           - "--data_volume=/cassandra_data"
           - "--internode_encryption=all"
-          - "--require_node_auth=true"
+          - "--require_node_auth=false"
```

Comment:
You need to re-enable node_auth, the changes here should be affecting how the certificates are handled, not modifying our security settings.